modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/config/PassThroughConfiguration.java

```diff
@@ -143,6 +143,11 @@ public int getMaximumConnectionLifespan() {
         return getIntProperty(PassThroughConfigPNames.MAXIMUM_CONNECTION_LIFESPAN, Integer.MAX_VALUE);
     }
 
+    public String getCorrelationHeaderName(){
```

Comment:
Shall we fix the formatting issue here?

Reply:
fixed in https://github.com/wso2/wso2-synapse/pull/1305